product_vision_and_principles.md

```diff
@@ -0,0 +1,65 @@
+- We will continue to run a smaller proportion of projects outside of these long term partnerships, but they will still be based on the same theory of change, i.e. generating shared meaning from consistent base maps, on which interactions can be designed that are tailored to the intervention need 
+
+The principle is that this theory of change needs to be reflected in our design decisions, particularly the distinction between "core platform features" for stakeholders and "customisable tools" for change agents, as described in the next principle.
+
+## Guiding Principle 2: Separate core from extensible functionality
+
+- Core platform features:
+	- should be at least as simple and intuitive to use for non-experts as Google Maps
+	- should be optimised, stable, and not change very much.
+	- have two primary use cases:
+		1. to help new folks understand how their organisational system/eco-system works and where they fit into it
+		2. to help everyday folks find organisational (as opposed to operational) information that would otherwise be hard to track down because it lives on a bunch of different corporate systems (e.g. Who owns this process? When does the contract for this IT system expire? What does this acronym stand for? Who else does this activity? etc.)
+	- should cover the following interactions and nothing else: search, map switching, map exploration, item selection, link navigation
+	- should also include an "advanced" mode for expert users who want to perform more advanced filters / queries directly on the model. This should include list mode, graph mode (?), filter tools, advanced search, detail level control. How this integrates with the basic experience is an ongoing design challenge; whatever choices we make must not jeopardise the simplicity of the experience for new users.
+- Customisable tools:
+	- should be based on a set of simple unified design principles / system, applied to an extensible set of discrete, configurable interaction "behaviours"
+	- have two primary use cases:
+		1. to allowing change agents to build "tasks" for users to perform in support of change intiatives
+		2. to configure custom states for specific user groups (e.g. a view with particular pre-set filters to be used as a decision support tool) This includes default states that could be applied to larger audiences e.g. having certain filter toggles always on
+
+## Guiding Principle 3: Promote autonomy down the user chain
+
+- The business model is based on the assumption that the use of the tool will develop organically over time, so we achieve our purpose best when we make it easy/desirable for people to move from one "level" of user to the next.
+- We can think of users in a "chain": from devs, to VM consultants, to "change agents", to "stakeholders". When we develop features, we should develop them in a way that maximises autonomy for the next user on in the chain. 
```

Comment:
I think that my vision about the user is different. User is the person who is not part of building the map or the platform. The user is only who consume the meaning. And if the end user can't "consume" then the stakeholder/client won't be interested in investing in such tool. Can you please add the end user here. :)